src/Init/Data/Int/DivMod/Lemmas.lean

```diff
@@ -2354,6 +2354,76 @@ theorem bmod_neg_bmod : bmod (-(bmod x n)) n = bmod (-x) n := by
   apply (bmod_add_cancel_right x).mp
   rw [Int.add_left_neg, ← add_bmod_bmod, Int.add_left_neg]
 
+/- ### ediv -/
```

Comment:
There are other ediv theorems higher up in the file. I feel these theorems may belong there.

Reply:
I tried to do this, but it would require a major restructuring of the file, as some theorems that are below that section are needed to prove the ones in this PR

Reply:
so I'd rather upstream this PR first and then consider moving the theorems, in a subsequent PR, if that can be useful